src/expr/src/scalar/func.rs

```diff
@@ -1205,6 +1205,48 @@ fn map_contains_key<'a>(a: Datum<'a>, b: Datum<'a>) -> Datum<'a> {
     }
 }
 
+fn map_contains_all_keys<'a>(a: Datum<'a>, b: Datum<'a>) -> Datum<'a> {
+    let map = a.unwrap_map();
+    let keys = b.unwrap_array();
+    if keys.elements().iter().any(|key| key.is_null()) {
+        // Map keys cannot be NULL.
+        return false.into();
+    }
```

Comment:
Wait, if they can't be null, why are we checking to see if they are?

Reply:
Maps cannot contain `NULL` keys, but the user can provide `NULL` keys values to the `?&` and `?|` operators. For example:
```
materialize=> SELECT '{1=>t, 2=>f}'::map[text=>bool] ?& ARRAY[NULL];
 ?column?
----------
 f
(1 row)
```

I think the change I just pushed made this a bit clearer, but please let me know if this still doesn't make sense!